data/hai/hai reveal 2 start.txt

```diff
@@ -283,14 +375,37 @@ mission "Hai Rescue: Meroot and Lucy"
 		log `Rescued a pair of researchers: a Hai woman named Meroot and a human woman named Lucy. They were kidnapped by men who brought them to Canyon. Returned them to <destination> where they now live together. They don't know the identities of the kidnappers.`
 		payment 315000
 		"hai slave prereq" ++
-		dialog `Meroot and Lucy thank you for your help and give you <payment>. Lucy decides to live in Hai space with Meroot, "It's safer here." The two don't know who the men were that kidnapped them.`
+		dialog `Meroot and Lucy thank you for your help and give you <payment>. Lucy decides to live in Hai space with Meroot. "It's safer here," she explains. You ask, but the two have no idea who the men that kidnapped them were.`
 	npc
 		government "Bounty Hunter that Won't Enter Hai Space"
 		personality plunders disables unconstrained
 		fleet "Large Core Pirates" 1
 		fleet "Small Core Pirates" 1
 			
 
+mission "Hai Rescue: Gipeep Hint"
+	landing
+	name "Lucrative Transport"
+	description "You've heard there's some suspiciously lucrative transport jobs on Maelstrom."
+	source
+		near "Nihal" 4
+		government "Republic" "Syndicate"
+		attributes "spaceport"
+		not attributes "station"
+	destination Maelstrom
+	to offer
+		"combat rating" > 50
+		has "Unwanted Cargo: done"
+		has "Hai Leaks Response 6: offered"
+		has "event: slightly lost"
+		not "Hai Rescue: Gipeep: offered"
+		not "event: hai-human treaty signed"
+		random < 80
+	on offer
+		conversation
+			`As you come in to land at the spaceport you listen to the usual chatter of captains coming and going. One guy is bragging particularly odiously about some lucrative transport jobs he'd picked up on <destination>. Of course it could mean nothing, but it's also very suspicious.`
```

Comment:
```suggestion
			`As you come in to land at the spaceport, you listen to the usual chatter of captains coming and going. One guy is bragging particularly odiously about some lucrative transport jobs he'd picked up on <planet>. Of course, it could mean nothing, but it's also very suspicious.`
```